### DISCUSSION TITLE
Shouldn't there be an option to not install bootloader?

### DISCUSSION THREAD

**Discussion by KoleckOLP** [archlinux/archinstall#2513] [General]
Yesterday I tried to install Arch on my laptop that already has Widnows 10 and Fedora Rawhide install, but I didn't watch arch to have it's own bootloarder I wanted to use the grub that I already have for fedora but arch install wouldn't let me install if I don't make a /boot partiton.

Should there be an option to install without bootloader, choosing none for bootloader and than using my own?

**Comment by Torxed** [comment]
Makes sense to be honest. 
It would be impressive if we could detect existing bootloaders and add an entry. 

But KISS says making it optional is good enough.

**Comment by svartkanin** [comment]
If it's optional then the installation can be started without choosing anything which I think will happen by accident occasionally.
To avoid that would it be fair to pre-populate the entry with systemd at least?

**Comment by SCRIPtRaven** [comment]
I'm going to Necro this purely because this is a good idea and we should have this. It comes in very handy for dual-booting two linux distributions. Sure you can do manual arch install, but then why have archinstall as a convenience. It is a must-have in my opinion

**Reply by svartkanin** [reply]
This has been implemented and is available https://github.com/archlinux/archinstall/pull/3677

Use --skip-boot
